### PR TITLE
Docs/readme.md

### PR DESCRIPTION
**Pull request by psycoder-sup**
Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/TBD-Labs-AI/latched/blob/reposetting/CONTRIBUTING.md),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

## changes

- Edit README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,105 @@
...
+
+Latched provides easy-to-use pipelines to perform ML models on various devices such as mobile, Nvidia jetson, Intel CPUs,  and accelerators.
+Latched covers both converting models and deploying them(Latched Model Manager, Latched Devices SDKs).
+
+Supported ML Tasks:
+
+📚 Text:
+
+- Small Language Models, to embed chat-bot or text analysis on device
+  - **Phi 3.5 mini** (iPhone 15 Pro)
```
Comment (sungmanch) [inline README.md:26]: I believe that we need to emphasize the combination of [Model, Compresson Method, Device]. At this point, I hope the representation like this: Phi-3.5-mini + OmniQuantW3A8 @ iPhone 15 Pro. 

Reference Image: https://privatellm.app/release-notes
image

File: README.md [hunk 2]
```diff
@@ -1,39 +1,105 @@
...
+  - Other models will be supported soon
+- Other tasks will be supported soon
+
+🏞️ VIsion:
```
Comment (sungmanch) [inline README.md:30]:
```suggestion
🏞️ Vision:
```

File: README.md [hunk 3]
```diff
@@ -1,39 +1,105 @@
...
-poetry run pre-commit run --all-files
-```
+
+
+<p align="center">
+    <img src="Images/Logo-no-background-3x.png" alt="Latched" width="50%">
+</p>
+
+<p align="center">
+    <a href="https://discord.gg/UxMXBHUWcr"><img src="https://img.shields.io/badge/community-5865F2?style=flat&logo=discord&logoColor=white" alt="Discord"/></a>
```
Comment (sungmanch) [inline README.md:8]: This is not important now, however, this seems better
image

URL: https://github.com/hatchet-dev/hatchet

File: README.md [hunk 4]
```diff
@@ -1,39 +1,105 @@
...
+   let model = LatchedModel(key: "model_key")
+   ```
+
+## Model Hub
+
+coming soon
+
+## Contributing
+
+Do you believe the future of AI is on edge computing? Do you want to make it happen?
```
Comment (sungmanch) [inline README.md:92]: Need to consider the accelerators. No need to keep my suggestion.

```suggestion
Do you believe the future of AI is on devices including mobile phone, robot, CCTV, accerlator? Do you want to make it happen?
```

**Review by sungmanch — COMMENTED** [review]
Please check my comments

File: README.md (hunk 1 above)
Comment (psycoder-sup) [reply README.md:26]: Checked

**Review by sungmanch — APPROVED** [review]
Thanks for the effort !

**Pull request merged**